apis/validator/duties/proposer.yaml

```diff
@@ -4,7 +4,19 @@ get:
     - Validator
   summary: "Get block proposers duties"
   operationId: "getProposerDuties"
-  description: "Request beacon node to provide all validators that are scheduled to propose a block in the given epoch"
+  description: "Request beacon node to provide all validators that are scheduled to propose a block in the given epoch.
+
+    Duties should only need to be checked once per epoch,
+    however a chain reorganization (of > MIN_SEED_LOOKAHEAD epochs) could occur,
```

Comment:
May be too early in the morning for me here, but is this condition correct?  I thought any chain reorganization > 0 epochs could alter the proposer duties.

Reply:
I think I screwed up and copy/pasted that from the attester version.